### PR TITLE
526 service history mg

### PR DESCRIPTION
**Pull request by markgreenburg**
- Updates the `servicePeriods` schema with enums instead of free text.

### CODE REVIEW

File: src/schemas/21-526EZ/schema.js [hunk 1]
```diff
@@ -331,7 +309,34 @@ let schema = {
...
+                  'Army',
+                  'Army Reserves',
+                  'Coast Guard',
+                  'Coast Guard Reserves',
+                  'Marine Corp',
+                  'Marine Corps Reserves',
+                  'Navy',
+                  'Navy Reserves',
+                  'NOAA',
+                  'USPHS',
```
Comment (cehsu) [inline src/schemas/21-526EZ/schema.js]: Should we spell these acronyms out and provide the acronyms in parentheses instead?

Comment (markgreenburg) [reply src/schemas/21-526EZ/schema.js]: Good call. I'm going to double check with Alex.

File: src/schemas/21-526EZ/schema.js [hunk 2]
```diff
@@ -331,7 +309,34 @@ let schema = {
...
+                  'National Oceanic and Atmospheric Administration',
+                  'Navy',
+                  'Navy Reserve',
+                  'Public Health Service'
+                ]
+              },
+              dateRange: {
+                $ref: '#/definitions/dateRangeAllRequired'
+              }
+            }
```
Comment (cvalarida) [inline src/schemas/21-526EZ/schema.js]: Should we keep the `minItems`, `maxItems`, and required fields, or were these causing issues in pre-fill?

Comment (markgreenburg) [reply src/schemas/21-526EZ/schema.js]: Yes, we should definitely keep them. 🤦‍♂️

**Pull request merged**